packages/ckeditor5-engine/tests/view/observer/selectionobserver.js

```diff
@@ -162,6 +162,46 @@ describe( 'SelectionObserver', () => {
 		changeDomSelection();
 	} );
 
+	it( 'should not pass non-valid object on Firefox', done => {
```

Comment:
I suggested simplification of the try/catch code but this makes this test fail. Do you think this test case is valid? If so, then how?